docs/content/en/docs/Development/debugging-station.md

```diff
@@ -0,0 +1,247 @@
+  namespace: kube-system
+spec:
+    serviceUUID: "ssh"
+    secretRef: "ssh-entanglement"
+    host: "127.0.0.1"
+    port: "22"
+    hostNetwork: true
+```
+
+{{% alert title="Note" color="warning" %}}
+
+If you have already a kubernetes cluster, you can install the [Entangle](/docs/reference/entangle) chart and just apply the manifest.
+
+{{% /alert %}}
+
+This entanglement will expose the port `22` in the node over the mesh network with the `ssh` service UUID so we can later connect to it. Replace the generated token in the `network_token` field in the Secret (check out the [documentation](/docs/reference/entangle) for advanced usage).
+
+In order to deploy the `Entanglement` automatically, we can add it to the `k3s` manifests folder in the cloud config file:
+
+```yaml
+write_files:
+- path: /var/lib/rancher/k3s/server/manifests/expose-ssh.yaml
+  permissions: "0644"
+  owner: "root"
+  content: |
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: ssh-entanglement
+        namespace: kube-system
+      type: Opaque
+      stringData:
+        network_token: ___GENERATED TOKEN HERE___
+      ---
+      apiVersion: entangle.kairos.io/v1alpha1
+      kind: Entanglement
+      metadata:
+        name: ssh-entanglement
+        namespace: kube-system
+      spec:
+         serviceUUID: "ssh"
+         secretRef: "ssh-entanglement"
+         host: "127.0.0.1"
+         port: "22"
+         hostNetwork: true
+```
+
+Here's an example of a complete cloud configuration file which automatically install a Kairos node in the bigger disk, and exposes ssh with `entangle`:
+
+```yaml
+#cloud-config
+
+install:
+ device: "auto"
+ auto: true
+ reboot: true
+
+hostname: debugging-station-{{ trunc 4 .MachineID }}
+
+users:
+- name: kairos
+  passwd: kairos
+  ssh_authorized_keys:
+  - github:mudler
+
+k3s:
+  enabled: true
+
+# Specify the bundle to use
+bundles:
+- targets:
+  - run://quay.io/kairos/community-bundles:system-upgrade-controller_latest
+  - run://quay.io/kairos/community-bundles:cert-manager_latest
+  - run://quay.io/kairos/community-bundles:kairos_latest
+
+kairos:
+  entangle:
+    enable: true
+
+write_files:
+- path: /var/lib/rancher/k3s/server/manifests/expose-ssh.yaml
+  permissions: "0644"
+  owner: "root"
+  content: |
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: ssh-entanglement
+        namespace: kube-system
+      type: Opaque
+      stringData:
+        network_token: b3RwOgogIGRodDoKICAgIGludGVydmFsOiA5MDAwCiAgICBrZXk6IER5NVRhdksxYzVGcE41YnVIZVhTWTRWejNEbDRRWFdHbnlIYzc5TzlPNGwKICAgIGxlbmd0aDogNDMKICBjcnlwdG86CiAgICBpbnRlcnZhbDogOTAwMAogICAga2V5OiBKNUZNMUpDVjNoN28wSXRtb3pPN3ZVcURrMVN0c3RNSTVBSm1XUGFSUUlNCiAgICBsZW5ndGg6IDQzCnJvb206IGU0TTNzZGQ5QzBCT2RCZkxvenBmaGNZZlFiNzQ3cnlBaVM5dWwyaXdPQ2kKcmVuZGV6dm91czogMFJuUTNEWWRCcmM5ekF0Rjl5djdxUU13WVYxTlJ4bW9CS1paN1l4MVdWRgptZG5zOiB0QktoMFJiNG04cXgzRTFJWTBJRDhpZGx6ZU1DU1lBOUR0MkF5TGRsQmUzCm1heF9tZXNzYWdlX3NpemU6IDIwOTcxNTIwCg==
```

Comment:
To me it's clear the this is just an example and a fresh token should be used but I wonder if we should replace this with a placeholder (also in other places below), to avoid people copying and pasting the same token.

Reply:
:+1: amended!